src/components/Button/Button.style.ts

```diff
@@ -46,11 +46,11 @@ const Outlined = styled(Button).attrs(props => ({
 `;
 
 export const ButtonPrimary = styled(Outlined)`
-  color: ${theme.colors.green};
+  color: ${COLOR_MAP.GREEN.BASE};
 `;
 
 export const ButtonSecondary = styled(Outlined)`
-  color: ${theme.colors.lightBlue};
```

Comment:
I think we don't use these buttons anywhere, we can probably remove them. Also, should it be `${props => props.theme.colors.green}`?

Reply:
^ please ignore 😅